public_app/components/SearchBox.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import axios from 'axios';
+
+
+class SearchBox extends React.Component {
+    constructor (props) {
+        super(props);
+        this.state = {
+            items: [],
+            id: '',
+            searchInput: '',
+            searchTitle: [],
+            searchYear: []
+        };
+        this.handleSearch = this.handleSearch.bind(this);
+
+        axios.get("api/movies").then((response) => {
+            this.setState({
+                items: response.data
+            });
+        });
+    }
+    handleSearch(value){
+//        this.setState({
+//            searchYear: this.state.items.filter((filt) =>{
```

Comment:
If you would commit every hour or two when the code works you would have all the previous version stored in github so there would be no need for comments